packages/cadl-ranch-specs/http/azure/resource-manager/models/resources/toplevel.tsp

```diff
@@ -211,4 +220,19 @@ interface TopLevelTrackedResources {
     ```
     """)
   listBySubscription is ArmListBySubscription<TopLevelTrackedResource>;
+
+  @scenario
+  @scenarioDoc("""
+      Resource notify operation.
+      Expected path: /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.ResourceManager.Models.Resources/topLevelTrackedResources/top/operation
+      Expected query parameter: api-version=2023-12-01-preview
+      Expected request body:
+      ```json
+      {
+        "message": "Send a operation to top level.",
+        "urgent": true
+      }
+      ```
+    """)
+  operation is ArmResourceActionNoContentSync<TopLevelTrackedResource, NotificationDetails>;
```

Comment:
`operation` seems too generic. Maybe `actionSync` to indicate the operation type.

Reply:
Fixed the action name from `operation` to `actionSync `

Reply:
Please also update the `scenarioDoc`. "Resource action" should be fine. Or "Resource sync action".